src/main/java/entity/Entity.java

```diff
@@ -52,6 +53,10 @@ public boolean isVisible() {
 		return visible;
 	}
 
+	public Vec2f getXY() {
```

Comment:
getLocation maybe?

Reply:
if you want but that will take up a lot of space